rollup.config.js

```diff
@@ -6,18 +6,27 @@ import commonjs from 'rollup-plugin-commonjs';
 import { terser } from 'rollup-plugin-terser';
 
 const BASE_PATH = process.cwd();
-const directory = 'dist/rulesets/oas/functions';
-const targetDir = path.join(BASE_PATH, directory);
 
 const functions = [];
-for (const file of fs.readdirSync(targetDir)) {
-  const targetFile = path.join(targetDir, file);
-  const stat = fs.statSync(targetFile);
-  if (!stat.isFile()) continue;
-  const ext = path.extname(targetFile);
-  if (ext !== '.js') continue;
-
-  functions.push(targetFile);
+
+const builtIns = ['oas', 'aas']
```

Comment:
We're kinda stuck with `oas` due to it being chosen a while back and not being worthwhile to rename it just yet, but for new things lets call it a name which people will recognize. I don't think aas is a thing, so let's just call it `asyncapi`.

```suggestion
const builtIns = ['oas', 'asyncapi']
```